ci/doc/how-to-setup-taos-ci-server.md

```diff
@@ -101,23 +101,30 @@ The file itself is sourced by a shell script, so it is required that the file co
 For more details, refer to http://manpages.ubuntu.com/manpages/trusty/man5/pbuilderrc.5.html
 ```bash
 $ vi /etc/pbuilderrc
-# man 5 pbuilderrc
+# If you want to see more details, please run 'man 5 pbuilderrc' command.
 DISTRIBUTION=xenial
 OTHERMIRROR="deb http://archive.ubuntu.com/ubuntu xenial universe multiverse |deb [trusted=yes] http://[id]:[password]@[your-own-server]/tools/ubuntu16.04/ /"
+$
+$ chown -R www-data:www-data /var/cache/pbuilder
+$
 $ sudo vi /etc/crontab
-#### Update pdebuild/pbuilder to keep latest apt repositories, /var/cache/pbuilder/base.tgz
+## Update a base Ubuntu image (e.g., /var/cache/pbuilder/base.tgz) of pdebuild/pbuilder to keep latest apt repositories.
 30 7 * * * root pbuilder update --override-config
 ```
+**(Optional)**: How to suppress a storage usage of /var/cache/pbuile folder
```

Comment:
To help  that administrator  can do a server optimization, Two optional techniques are updated/added.